tests/integration_tests.rs

```diff
@@ -74,7 +76,40 @@ async fn test_wascc_provider() {
 
     assert_eq!(pod.status.unwrap().phase.unwrap(), "Pending");
 
-    // TODO: check the pod status, wait for it to become ready, check logs
+    let inf: Informer<Pod> = Informer::new(
+        client,
+        ListParams::default()
+            .fields("metadata.name=hello-wascc")
+            .timeout(10),
+        Resource::namespaced::<Pod>("default"),
+    );
+
+    let mut watcher = inf
+        .poll()
+        .await
+        .expect("failed to poll for pod events")
+        .boxed();
+
+    while let Some(event) = watcher
+        .try_next()
+        .await
+        .expect("failed to poll for a pod event")
+    {
+        match event {
+            WatchEvent::Modified(o) => {
```

Comment:
There should probably be a timeout here so that if it doesn't reach running that it doesn't keep going forever

Reply:
There's a ListParams timeout set above! Does that satisfy it, or is there another timeout we need to set?

Reply:
Oh I skipped right over that part. I'm not used to list params having timeouts